tests/test_unittest/test_gtscript_frontend.py

```diff
@@ -241,6 +229,37 @@ def definition_func(inout_field: gtscript.Field[float]):
             )
 
 
+class TestAxisSyntax:
+    def test_good_syntax(self):
+        def definition_func(in_field: gtscript.Field[float], out_field: gtscript.Field[float]):
+            with computation(PARALLEL), interval(...):
+                out_field = in_field[I + 1, J - 1]
```

Comment:
I would also test indices without offset like just `I`, 0-offsets like `J + 0` or `K - 0` and offsets where the indices are not in the same order of the field (which I guess it should be supported) like `field[J-1,K+1,I]`